Cargo.toml

```diff
@@ -23,7 +23,7 @@ chrono-humanize = "0.2.1"
 dialoguer = { version = "0.10.0", features = ["fuzzy-select"] }
 clap = { version = "3.1", features = ["derive", "cargo"] }
 regex = "1.5.5"
-openssl = { version = "0.10.38", features = ["vendored"] }
```

Comment:
this is needed for osx cross compile :\ 

no way to do this as a patch in nix ?

(but let's see if it works first)

Reply:
There is ways to do it.. but in both case I can't compile it on Nix so far 🤔

Reply:
this is weird, do you have the full log by any chance?